lib/rspec/matchers/built_in/yield.rb

```diff
@@ -326,7 +337,7 @@ def negative_failure_reason
           end
```

Comment:
Ah, and because of that it's now failing since I've (locally) added test coverage for the messages as you requested.